frontend/src/views/plugin/Toast.test.js

```diff
@@ -0,0 +1,14 @@
+import { describe, it, expect } from "vitest";
+import Toast from "./Toast";
+
+describe("#Toast", () => {
+  it("should return a Swal object configured as a toast notification", () => {
+    const toast = Toast();
+
+    expect(toast.toast).toBe(undefined);
```

Comment:
Why they are `undefined`? 

Expected return values:
    toast: true,
    position: "top",
    showConfirmButton: false,
    timer: 1500,
    timerProgressBar: true,

Reply:
Have to double check with the docs!